app.js

```diff
@@ -1,162 +1,241 @@
 'use strict';
+
 var userInfo = {
   //temporary hard coded values
   "spiraUrl": "https://demo.spiraservice.net/rodrigo-pereira/",
   "username": "administrator",
   "apikey": "{AA50F584-BBC9-42A0-81BA-9F8A5CD8144A}",
+  "auth": "?username=administrator&api-key={AA50F584-BBC9-42A0-81BA-9F8A5CD8144A}",
```

Comment:
Yes, There is a section commented out that actually sets these values to be the values in the login input boxes (app.js 143)